ui/app/components/role-ssh-edit.js

```diff
@@ -5,4 +5,12 @@ export default RoleEdit.extend({
     this._super(...arguments);
     this.set('backendType', 'ssh');
   },
+
+  actions: {
+    updateTtl(path, val) {
+      const model = this.get('model');
+      let valueToSet = val.enabled === true ? `${val.seconds}s` : undefined;
```

Comment:
does value always come in seconds?  Just curious.

Reply:
Yes, by design that value is always in seconds because, while we allow users to set it in days, the backend is only able to parse `s`, `m`, or `h` and it gets translated to seconds anyway